src/common/components/play/multiplayer/LobbyStatus.js

```diff
@@ -31,16 +19,48 @@ const LobbyStatus = (props) => {
     <span>Connecting ...</span>
   );
 
+  const renderPlayerName = (userData, clientId) => userData && !userData.uid.startsWith('guest_') ? 
```

Comment:
nit: Put some parens around this function, yo

Reply:
Done